quiz/file_reader.py

```diff
@@ -0,0 +1,55 @@
+"""
+Module handle file operations
+"""
+import re
+
+from question import Question
+
+
+class FileReader:
+    """
+    Class to perform file operations
+    """
+    file_format = None
+
+    def __init__(self):
+        self.file_format = '^\d+[\.]\s.*';
+
+    @staticmethod
+    def read_file(file_path):
+        """
+        Read keys from file and return list of keys
+        """
+        file_content = None
+        try:
+            with open(file_path) as f:
+                file_content = f.read().splitlines()
+        except FileNotFoundError:
+            raise
+
+        return file_content
+
+    def get_questions(self):
+        """
+        Read questions from file and return list of questions
+        """
+        questions = []
+        try:
+            question_content = self.read_file("questions.txt")
+            key_content = self.read_file("key.txt")
+
+            for question, key in zip(question_content, key_content):
+                if self.validate_format(question) and self.validate_format(key):
```

Comment:
We shouldn't pass silently if a question/key format is not correct, an exception should be raised. 
`Raise Exception('File format is not correct, expected .... `)` etc

Reply:
@naeem91 made the changes and merged to dev branch.